src/main/java/seedu/address/model/applicant/Age.java

```diff
@@ -7,7 +7,7 @@
     public static final String MESSAGE_CONSTRAINTS =
             "Age should only contain numbers, and it should be at least 2 digits long";
     public static final String VALIDATION_REGEX = "\\d{2,}";
-    public final String age;
+    public final String value;
```

Comment:
Consider using a more descriptive word to denote the age? Perhaps "years" would be a good substitute?